bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -62,9 +63,9 @@ class Edge extends WebBrowser {
 	private static final int MAXIMUM_CREATION_RETRIES = 5;
 	private static final Duration MAXIMUM_OPERATION_TIME = Duration.ofMillis(Integer.getInteger(WEB_VIEW_OPERATION_TIMEOUT, 5_000));
 
-	private record WebViewEnvironment(ICoreWebView2Environment environment, ArrayList<Edge> instances) {
+	private record WebViewEnvironment(ICoreWebView2Environment environment, List<Edge> instances) {
```

Comment:
I wonder if a record is really useful here. can the instances probably be part of `ICoreWebView2Environment`?

Reply:
`ICoreWebView2Environment` is a native interface (https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2environment?view=webview2-1.0.2957.106), so I guess nothing we could (or at least should) extend.

Reply:
okay than its not an option of course ...